tests/tests_app/cli/test_cmd_install.py

```diff
@@ -61,11 +61,11 @@ def test_valid_unpublished_app_name():
 
 
 @pytest.mark.skip(reason="need to figure out how to authorize git clone from the private repo")
-def test_app_install(tmpdir):
+def test_app_install(tmpdir, monkeypatch):
     """Tests unpublished app install."""
 
     cwd = os.getcwd()
-    os.chdir(tmpdir)
+    monkeypatch.chdir(tmpdir)
```

Comment:
we should also remove the `os.chdir(...)` at the end of the test (in all the tests that have it). When using the monkeypatch, this teardown logic is no longer required.

Reply:
Right! Sorry I missed it.

Reply:
@awaelchli Thanks, I have removed it.

Reply:
Hi, are there any changes to make that I might have missed?

Reply:
I added a few more things that was missed. No big deal. Let's see if tests pass and then we should be good! Thanks

Reply:
Thanks for the changes! It's my first contribution to Lightning so I was a bit confused throughout 😅

Reply:
Perfectly normal 🎉  and we're here to help :)